diff -urN aa/ultimatestunts-src-0451/graphics/graphicworld.cpp ultimatestunts-src-0451/graphics/graphicworld.cpp
--- aa/ultimatestunts-src-0451/graphics/graphicworld.cpp	2004-02-20 10:15:58.000000000 +0100
+++ ultimatestunts-src-0451/graphics/graphicworld.cpp	2004-08-28 20:33:56.889061144 +0200
@@ -109,7 +109,7 @@
 CTexture **CGraphicWorld::getTextureSubset(CString indices)
 {
 	//printf("Indices: \"%s\"\n", indices.c_str());
-	CTexture **ret = new (CTexture *)[1+indices.length()/2]; //We don't need more
+	CTexture **ret = new CTexture *[1+indices.length()/2]; //We don't need more
 	int i=0;
 	while(true)
 	{
diff -urN aa/ultimatestunts-src-0451/simulation/world.cpp ultimatestunts-src-0451/simulation/world.cpp
--- aa/ultimatestunts-src-0451/simulation/world.cpp	2004-08-20 14:15:17.000000000 +0200
+++ ultimatestunts-src-0451/simulation/world.cpp	2004-08-28 20:33:36.645138688 +0200
@@ -307,7 +307,7 @@
 CMaterial **CWorld::getMaterialSubset(CString indices)
 {
 	//printf("Indices: \"%s\"\n", indices.c_str());
-	CMaterial **ret = new (CMaterial *)[1+indices.length()/2]; //We don't need more
+	CMaterial **ret = new CMaterial *[1+indices.length()/2]; //We don't need more
 	int i=0;
 	while(true)
 	{
diff -urN aa/ultimatestunts-src-0451/stunts3dedit/textureloader.cpp ultimatestunts-src-0451/stunts3dedit/textureloader.cpp
--- aa/ultimatestunts-src-0451/stunts3dedit/textureloader.cpp	2004-02-20 10:15:58.000000000 +0100
+++ ultimatestunts-src-0451/stunts3dedit/textureloader.cpp	2004-08-28 20:34:58.667669368 +0200
@@ -55,7 +55,7 @@
 
 	line=f.readl();  //# of textures
 	m_N = line.toInt();
-	m_TexArray = new (CTexture *)[m_N];
+	m_TexArray = new CTexture *[m_N];
 
 	for(int i=0; i<m_N; i++) //textures
 	{
