--- ultimatestunts-srcdata-0751/configure.ac.orig	2008-06-08 19:20:59.000000000 +0200
+++ ultimatestunts-srcdata-0751/configure.ac	2008-07-31 00:15:44.000000000 +0200
@@ -255,7 +255,7 @@
 dnl fi
 
 dnl The location of the data files:
-usdatadir='${datadir}/ultimatestunts/'
+usdatadir='${datadir}/games/ultimatestunts/'
 
 dnl The configure-time values of these variables:
 dnl sysconfdir_configure_time=`eval eval eval eval echo "${sysconfdir}"`
