--- ultimatestunts-srcdata-0561/configure.ac	2005-09-08 12:12:18.000000000 +0200
+++ ultimatestunts-srcdata-0561/configure.ac.new	2005-10-16 02:18:08.000000000 +0200
@@ -183,7 +183,7 @@
 dnl ------------------
 dnl The directory locations
 dnl ------------------
-usdatadir="${prefix}/share/ultimatestunts/"
+usdatadir="${datadir}/games/ultimatestunts/"
 
 AC_DEFINE_UNQUOTED(PREFIXDIR, "$prefix")
 
